raiden_contracts/constants.py

```diff
@@ -8,7 +8,7 @@
 from raiden_contracts.utils.type_aliases import ChainID, Locksroot
 
 # The last digit is supposed to be zero always. See `RELEASE.rst`.
-CONTRACTS_VERSION = "0.37.0"
+CONTRACTS_VERSION = "0.38.0"
 ALDERAAN_VERSION = "0.37.0"
```

Comment:
I guess we should add - for completeness sake - a `BESPIN_VERSION` as well.

Reply:
Sounds ok, but it should have the same value as ALDERAAN, right?

Reply:
Yes

Reply:
Done.